app/population/controllers/edit.test.js

```diff
@@ -215,15 +215,17 @@ describe('Population controllers', function () {
       })
     })
 
-    describe('setButtonText', function () {
+    describe('setCancelUrl', function () {
```

Comment:
`setButtonText` was completely removed, and `setPageTitle` was added below - but the git diff has become confused and thinks there were bigger changes than actually occurred.